sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubConfigurationTests.cs

```diff
@@ -124,7 +161,7 @@ private EventHubOptions CreateOptionsFromConfig()
                 { $"{extensionPath}:LoadBalancingUpdateInterval", "00:00:21" },
                 { $"{extensionPath}:LoadBalancingStrategy", "0" },
                 { $"{extensionPath}:InitialOffsetOptions:Type", "FromEnqueuedTime" },
-                { $"{extensionPath}:InitialOffsetOptions:EnqueuedTimeUTC", "2020-09-13T12:00Z" },
+                { $"{extensionPath}:InitialOffsetOptions:EnqueuedTimeUTC", "2020-09-13 12:00:00Z" },
```

Comment:
Why did the format change? Can we handle the old format?

Reply:
Yes we can handle the old format, but I couldn't figure out how to round-trip it so the values would match in the tests.

Reply:
Let's add some tests that verify we can read old values as well.